__tests__/core/validation/validators/UrlValidator.spec.ts

```diff
@@ -0,0 +1,74 @@
+    test('https://', () => {
+      expect(UrlValidator.validate('http://')).toBeFalsy()
+    })
+    test('https://foo', () => {
+      expect(UrlValidator.validate('https://q')).toBeFalsy()
+    })
+    test('https://foo..', () => {
+      expect(UrlValidator.validate('https://foo..')).toBeFalsy()
+    })
+    test('https://foo.-', () => {
+      expect(UrlValidator.validate('https://foo.-')).toBeFalsy()
+    })
+    test('https://foo--', () => {
+      expect(UrlValidator.validate('https://foo--')).toBeFalsy()
+    })
+    test('https://192.0.0', () => {
+      expect(UrlValidator.validate('https://foo--')).toBeFalsy()
+    })
+  })
+  describe('return true when passing in', () => {
+    test('google.com', () => {
+      expect(UrlValidator.validate('google.com')).toBeTruthy()
+    })
+    test('http://google.com', () => {
+      expect(UrlValidator.validate('http://google.com')).toBeTruthy()
+    })
+    test('https://google.com', () => {
+      expect(UrlValidator.validate('https://google.com')).toBeTruthy()
+    })
+    test('https://goo-gle.com', () => {
+      expect(UrlValidator.validate('https://goo-gle.com')).toBeTruthy()
+    })
+    test('https://goo.gle.com', () => {
+      expect(UrlValidator.validate('https://goo.gle.com')).toBeTruthy()
+    })
+    test('192.0.0.1:8080', () => {
+      expect(UrlValidator.validate('192.0.0.1:8080')).toBeTruthy()
+    })
+    test('localhost:8080', () => {
+      expect(UrlValidator.validate('localhost:8080')).toBeTruthy()
+    })
+    test('localhost', () => {
+      expect(UrlValidator.validate('localhost')).toBeTruthy()
+    })
+    test('http://api-01.eu-west-1.0941-v1.symboldev.network:3000', () => {
+      expect(UrlValidator.validate('http://api-01.eu-west-1.0941-v1.symboldev.network:3000')).toBeTruthy()
+    })
+    test('http://api-02.ap-northeast-1.0941-v1.symboldev.network:3000', () => {
+      expect(UrlValidator.validate('http://api-02.ap-northeast-1.0941-v1.symboldev.network:3000')).toBeTruthy()
+    })
+    test('http://api-01.ap-northeast-1.0941-v1.symboldev.network:3000', () => {
+      expect(UrlValidator.validate('http://api-01.ap-northeast-1.0941-v1.symboldev.network:3000')).toBeTruthy()
+    })
+    test('http://api-01.eu-central-1.0941-v1.symboldev.network:3000', () => {
+      expect(UrlValidator.validate('http://api-01.eu-central-1.0941-v1.symboldev.network:3000')).toBeTruthy()
+    })
+    test('http://api-02.eu-central-1.0941-v1.symboldev.network:3000', () => {
+      expect(UrlValidator.validate('http://api-02.eu-central-1.0941-v1.symboldev.network:3000')).toBeTruthy()
+    })
+    test('http://api-01.ap-southeast-1.0941-v1.symboldev.network:3000', () => {
+      expect(UrlValidator.validate('http://api-01.ap-southeast-1.0941-v1.symboldev.network:3000')).toBeTruthy()
+    })
+    test('http://api-01.us-west-1.0941-v1.symboldev.network:3000', () => {
+      expect(UrlValidator.validate('http://api-01.us-west-1.0941-v1.symboldev.network:3000')).toBeTruthy()
+    })
```

Comment:
I think only one of these URLS should be enough

Reply:
> I think only one of these URLS should be enough

I've considered using only one of them before, but in order to ensure that all our current nodes can be connected, there will be no other accidents

Reply:
Then maybe importing nodes that are in the network configuration object would be better? Nodes URLs will bet changed many times, if this test file is not maintained with up-to-date URLs, it won't really make sense anymore in a close future

Reply:
Even if one day these URLs are out of date, these out of date URL unit tests still make sense

Reply:
there is no need to copy all these URLs in there, please remove them and use only one.